libraries/AP_HAL_ChibiOS/hwdef/CORVON743V1/hwdef.dat

```diff
@@ -0,0 +1,179 @@
+# hw definition file for processing by chibios_hwdef.py
+# for the CORVON743V1 hardware
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# board ID. See Tools/AP_Bootloader/board_types.txt
+APJ_BOARD_ID AP_HW_CORVON743V1
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+# ChibiOS system timer
+STM32_ST_USE_TIMER 12
+define CH_CFG_ST_RESOLUTION 16
+
+# flash size
+FLASH_SIZE_KB 2048
+FLASH_RESERVE_START_KB 128
+
+define HAL_STORAGE_SIZE 32768
+STORAGE_FLASH_PAGE 14
+
+#BATTERY MONITOR ON
+BATT_MONITOR 4
+
+#enable second OSD type
+OSD_TYPE2 = 5
+
+#Serial Port defaults
+SERIAL2_PROTOCOL 42
```

Comment:
incorrect define....add the appropriate defines for each serial port:
define DEFAULT_SERIAL5_PROTOCOL SerialProtocol_RCIN
define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_MSP_DisplayPort
define DEFAULT_SERIAL6_PROTOCOL SerialProtocol_ESCTelemetry
define DEFAULT_SERIAL4_PROTOCOL SerialProtocol_MAVLink2